pyteal/ast/subroutine_test.py

```diff
@@ -1302,3 +1302,112 @@ def mySubroutine(a1, a2, a3, a4, a5, a6, a7, a8, a9, a10):
         actual, _ = declaration.__teal__(options)
         options.setSubroutine(None)
         assert actual == expected
+
+
+def test_docstring_parsing_with_different_format():
+    @pt.ABIReturnSubroutine
+    def google_style(a: pt.abi.Uint64, *, output: pt.abi.Uint64):
+        """
+        Example of a ABIReturnSubroutine with Google format docstring.
+
+        Args:
+            a: an abi Uint64 value
+            output: an abi Uint64 value get set from argument a
+
+        Returns:
+            A PyTeal expression that sets output Uint64 value as argument a.
+        """
+        return output.set(a)
+
+    mspec_dict = google_style.method_spec().dictify()
+
+    assert (
+        mspec_dict["desc"]
+        == "Example of a ABIReturnSubroutine with Google format docstring."
+    )
+    assert mspec_dict["args"][0]["desc"] == "an abi Uint64 value"
+    assert (
+        mspec_dict["returns"]["desc"]
+        == "A PyTeal expression that sets output Uint64 value as argument a. "
+        + "an abi Uint64 value get set from argument a"
+    )
+
+    @pt.ABIReturnSubroutine
+    def epy_style(a: pt.abi.Uint64, *, output: pt.abi.Uint64):
+        """
+        Example of a ABIReturnSubroutine with Google format docstring.
```

Comment:
nit: change docstring to 
```suggestion
        Example of a ABIReturnSubroutine with epytext format docstring.
```